src/ImageSharp.Web/Commands/QueryCollectionRequestParser.cs

```diff
@@ -12,18 +11,24 @@ namespace SixLabors.ImageSharp.Web.Commands
     /// <summary>
     /// Parses commands from the request querystring.
     /// </summary>
-    public class QueryCollectionRequestParser : IRequestParser
+    public sealed class QueryCollectionRequestParser : IRequestParser
     {
         /// <inheritdoc/>
         public IDictionary<string, string> ParseRequestCommands(HttpContext context)
         {
-            if (!context.Request.Query.Any())
+            if (context.Request.Query.Count == 0)
             {
                 return new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
             }
 
-            return QueryHelpers.ParseQuery(context.Request.QueryString.ToUriComponent())
-                               .ToDictionary(k => k.Key, v => v.Value.ToString());
+            var parsed = QueryHelpers.ParseQuery(context.Request.QueryString.ToUriComponent());
```

Comment:
It's hard to tell what the returned type of `parsed` is here, can we use the explicit declaration.